File: Pertemuan 5/logico.py
a = input("Masukkan Nilai A = ")
b = input("Masukkan Nilai B = ")

print("A == B = "+str(a==b))
print("A != B = "+str(a!=b))
print("A > B = "+str(a>b))
print("A < B = "+str(a<b))
print("A <= B = "+str(a<=b))
print("A >= B = "+str(a>=b))

x = int(a)<5 and int(b)>4
y = int(a)<5 or int(b)>4
print("A < 5 and B > 4 = "+str(x))
print("A < 5 or B > 4 = "+str(y))
print("A < 5 and(not) B > 4 = "+str(not(int(a<b))))

File: Pertemuan 5/Logico.java
import java.util.Scanner;

public class Logico {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);

        System.out.print("Masukkan Nilai A = ");
        double a = console.nextDouble();

        System.out.print("Masukkan Niali B = ");
        double b = console.nextDouble();

        System.out.println("A == B = "+(a==b));
        System.out.println("A != B = "+(a!=b));
        System.out.println("A > B = "+(a>b));
        System.out.println("A < B = "+(a<b));
        System.out.println("A >= B = "+(a>=b));
        System.out.println("A <= B = "+(a<=b));

        boolean hasil1 = (a<5 && b>2);
        boolean hasil2 = (a<5 || b>2);
        System.out.println("Hasil dari A < 5 && B > 2 = "+ hasil1);
        System.out.println("Hasil dari A < 5 || B > 2 = "+ hasil2);
    }
}